src/SDK.ts

```diff
@@ -413,16 +413,16 @@ export default class SDK implements ISDK {
    * @param requestId: RequestId to use for session init.
    * @param queueAvailabilityOptionalParams: Optional parameters for session init.
    */
-  public async getQueueAvailability(requestId: string, queueAvailabilityOptionalParams: IGetQueueAvailabilityOptionalParams = {}): Promise<QueueAvailability> {
+  public async getAgentAvailability(requestId: string, queueAvailabilityOptionalParams: IGetQueueAvailabilityOptionalParams = {}): Promise<QueueAvailability> {
```

Comment:
I'm okay with renaming this method to reflect the new API change. However, I'd recommend to kept the old method to prevent breaking changes on components depending the older method if they uptake the newer version of the SDK.

```
public async getQueueAvailability(requestId: string, queueAvailabilityOptionalParams: IGetQueueAvailabilityOptionalParams = {}): Promise<QueueAvailability> {
    console.warn("getQueueAvailability() will be deprecated in an upcoming release. Please use getAgentAvailability() instead. ");
    return getAgentAvailability(requestId, queueAvailabilityOptionalParams);
}
```

Reply:
These are publicly available methods and we should always follow proper deprecation policy. 
Please introduce a new method with this name and call the old one deprecated. The old one should be removed after 2 months.

Reply:
Or provide a confirmation from telemtry that the previous one has no usage.

Reply:
This was a private feature and is not used by any Cx today in PROD. FCB is also disabled by default. Only used by testing team for internal validation

lcw_ocsdkevents
| where Event contains "GetQueueAvailability"
//| where EventInfo_Time > ago(1d)
//| where OrganizationId == "360bcd92-2d79-490f-935a-52bc6c6dbdbf"
| summarize count() by OrganizationId


e0d4d65d-28f4-41c6-b4d6-c2e5e226f74d-CustomerTest
360bcd92-2d79-490f-935a-52bc6c6dbdbf -Consultredesign
3f50e0e7-b6c4-49fc-9db6-008472a04663- CustomerTest
21ac8b0c-19bf-49aa-a7b3-b11e0e81fe87 - Customertest